CoreScriptsRoot/Modules/BackpackScript3D.lua

```diff
@@ -0,0 +1,266 @@
+local ICON_SIZE = 48
+local ICON_SPACING = 52
+local PIXELS_PER_STUD = 64
+
+local SLOT_BORDER_SIZE = 0
+local SLOT_BORDER_HOVER_SIZE = 4
+local SLOT_BORDER_COLOR = Color3.new(90/255, 142/255, 233/255)
+
+local HOPPERBIN_ANGLE = math.rad(-45)
+local HOPPERBIN_ROTATION = CFrame.Angles(HOPPERBIN_ANGLE, 0, 0)
+local HOPPERBIN_OFFSET = Vector3.new(0, 0, -5)
+
+local Tools = {}
+local ToolsList = {}
+
+local BackpackScript = {}
+local topbarEnabled = false
+
+local player = game.Players.LocalPlayer
+local CoreGui = game:GetService('CoreGui')
+
+local hopperbinPart = Instance.new("Part", workspace.CurrentCamera)
```

Comment:
I just want to note here that at some point we will want to find a better place for these parts to live than Camera